sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithStateTTLTest.scala

```diff
@@ -143,43 +143,54 @@ abstract class TransformWithStateTTLTest
         AddData(inputStream, InputEvent("k1", "put", 1)),
         // advance clock to trigger processing
         AdvanceManualClock(1 * 1000),
+        // In the primary index, we should have that k1 -> [1].
+        // The TTL index has (6100, k1) -> empty. The min index has k1 -> 61000.
```

Comment:
nit: TTL index and min-expiry index are off? Looks like this happens in below code comment as well.